lib/discovery.js

```diff
@@ -16,7 +16,7 @@ module.exports = mixinDiscovery;
 function mixinDiscovery(DB2, db2) {
   var async = require('async');
 
-DB2.prototype.paginateSQL = function(sql, orderBy, options) {
+  DB2.prototype.paginateSQL = function(sql, orderBy, options) {
```

Comment:
Why are these all DB2.prototype instead of Dashdb.prototype .. I'd really like these to be consistent with the connector they are in.